filetests/isa/intel/binary64.cton

```diff
@@ -145,6 +145,149 @@ ebb0:
     ; asm: movq %rcx, %r10
     [-,%r10]             v112 = copy v1          ; bin: 49 89 ca
 
+    ; Load/Store instructions.
+
+    ; Register indirect addressing with no displacement.
+
+    ; asm: movq %rcx, (%rsi)
+    store v1, v2                                ; bin: 48 89 0e
+    ; asm: movq %rsi, (%rcx)
+    store v2, v1                                ; bin: 48 89 31
+    ; asm: movl %ecx, (%rsi)
+    istore32 v1, v2                             ; bin: 40 89 0e
+    ; asm: movl %esi, (%rcx)
+    istore32 v2, v1                             ; bin: 40 89 31
+    ; asm: movw %cx, (%rsi)
+    istore16 v1, v2                             ; bin: 66 40 89 0e
+    ; asm: movw %si, (%rcx)
+    istore16 v2, v1                             ; bin: 66 40 89 31
+    ; asm: movb %cl, (%rsi)
+    istore8 v1, v2                              ; bin: 40 88 0e
+    ; asm: movb %sil, (%rcx)
+    istore8 v2, v1                              ; bin: 40 88 31
+
+    ; asm: movq (%rcx), %rdi
+    [-,%rdi]            v460 = load.i64 v1      ; bin: 48 8b 39
+    ; asm: movq (%rsi), %rdx
+    [-,%rdx]            v461 = load.i64 v2      ; bin: 48 8b 16
```

Comment:
I think it's best to keep the value numbers monotonically increasing, if possible.